fhir-ig-us-core/CHANGELOG

```diff
@@ -0,0 +1 @@
+Changed constraint provenance-1 in StructureDefinition-us-core-provenance.json from: ($this.agent.who.resolve().is Practitioner or Device) implies exists() to: ($this.agent.who.resolve().is(Practitioner) or $this.agent.who.resolve().is(Device)) implies exists() to correct a FHIRPath syntax error
```

Comment:
Thanks for documenting what you did to change it.
We also have https://ibm.github.io/FHIR/Conformance#hl7-fhir-r4-v401-errata as a place to capture erratta like this, but presently its empty.
Do you think it makes sense to find all our little CHANGELOG things like this and link to them from there?

Reply:
I'd also like to get into the habit of including a link to a corresponding ticket from jira.hl7.org ...do we have one for this yet?

Reply:
I don't have Jira issue yet but should have one today.

Reply:
Opened issue: https://jira.hl7.org/browse/FHIR-27065